src/tools/Repl/Program.cs

```diff
@@ -361,21 +361,38 @@ private static string PrintResult(FormulaValue value, bool minimal = false)
                     resultString = "<table>";
                 }
                 else
-                {
-                    var columnWidth = new int[table.Rows.First().Value.Fields.Count()];
+                {                    
+                    var columnCount = 0;
+                    foreach (var row in table.Rows)
+                    {
+                        if (row.Value != null)
+                        {
+                            columnCount = Math.Max(columnCount, row.Value.Fields.Count());
+                        }
+                    }
+
+                    if (columnCount == 0)
```

Comment:
>if (columnCount == 0) [](http://example.com/codeflow?start=20&length=21)

Instead of checking columnCount (possibly record with 0 fields), we should check DValue.IsBlank.